docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -107,20 +75,35 @@ spec:
       vlan: 22
 EOF
 ----
++
+* In case when `neutron-sriov-nic-agent` is running on the existing Compute nodes, physical device mappings needs to be checked and set the same in the
+`OpenStackDataPlaneNodeSet` custom resource (CR). Those options will need to be set in the `OpenStackDataPlaneNodeSet` CR. For more information, see xref:reviewing-the-openstack-control-plane-configuration_adopt-control-plane[Reviewing the {rhos_prev_long} control plane configuration].
+
+.Variables
+
+Define the shell variables used in the Fast-forward upgrade steps below.
```

Comment:
So, is *Variables* going to be a "convention" used in this guide?

I would be tempted to make *Variables* a bullet item under prerequisites:

````
- Define the shell variables necessary to run the script runs the Fast-forward upgrade.
  
  <code snippet>
  
  o Replace the value for FIP with the IP address of the test instance on the source cloud.
  o Replace ["standalone.localdomain"]="192.168.122.100" with the name of the Compute
      node and its IP address.
````

Reply:
Yeah I've been doing .Variables in each procedure. I know we haven't done that before, but maybe it's never come up. I agree that it sounds like a prerequisite and wasn't sure if it should be called out separately. I don't know if I should make the call on that or ask for more guidance from our team.